packages/components/psammead-figure/src/index.stories.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withReadme } from 'storybook-readme';
+import Image from '@bbc/psammead-image';
+import Readme from '../README.md';
+import Figure from '.';
+
+const imageAlt =
+  'Map of the UK displaying Syrian refugees and asylum seekers per 10000 population. Ranges from 0 to 17.';
+const imageSrc =
+  'https://ichef.bbci.co.uk/news/640/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png';
+const imageWidth = 853;
+storiesOf('Figure', module)
```

Comment:
I'm scared this'll sound pedantic, but given we're showcasing multiple things at once here, can the story name reflect that?

"Figure containing image", for example. It also allows for/implies a future where we showcase the use of Figure with other types of children -- table, code snippet, etc. which seems more in line with the semantic meaning of the HTML figure tag.

Reply:
Good point - I'll update that 👍